### ISSUE TITLE
[Feature Request] retry failed upload

### ISSUE THREAD

**Issue by Aokromes** [telegramdesktop/tdesktop#26905]
### Is your feature request related to a problem?

sometimes one upload fails and you must restart from start, a good idea can be to automatically resume it

### Describe the solution you'd like

automatically resume failed upload

### Describe alternatives you've considered

allow user resume failed upload

**Comment by Aokromes** [comment]
also, if you upload 10 files and 1 fails and you miss it failed the whole upload will be stuck waiting to cancel the failed upload..

**Comment by buttler2** [comment]
I'm the same, Aokromes is right. Implement this improvement

**Comment by ElektroStudios** [comment]
It is incredible that this basic and essential feature that was requested almost a thousand times, it still ignored after many years.

**Comment by john-preston** [comment]
@ElektroStudios no time, sorry 🤷

**Comment by Alphos2000** [comment]
Yes please. I'm tired deleting a group of ten files just because one of them failed and have to reupload all of them again. Just an option to try again without losing the order in chat would be awesome.

**Comment by Aokromes** [comment]
also, one anoying problem of this, is that failed file doesn't warns there are 9 pending files to send to telegram and if you close telegram you will need to reupload the 10 files.
